### ISSUE TITLE
About the finetuning of self-training

### ISSUE THREAD

**Issue by Serge-weihao** [yzou2/cbst#6]
I used your command of 
"python issegm/solve_AO.py --num-round 6 --test-scales 1850 --scale-rate-range 0.7,1.3 --dataset gta --dataset-tgt cityscapes --split train --split-tgt val --data-root DATA_ROOT_GTA5 --data-root-tgt DATA_ROOT_CITYSCAPES --output gta2city/cbst --model cityscapes_rna-a1_cls19_s8 --weights models/gta_rna-a1_cls19_s8_ep-0000.params --batch-images 2 --crop-size 500 --origin-size-tgt 2048 --init-tgt-port 0.15 --init-src-port 0.03 --seed-int 0 --mine-port 0.8 --mine-id-number 3 --mine-thresh 0.001 --base-lr 1e-4 --to-epoch 2 --source-sample-policy cumulative --self-training-script issegm/solve_ST.py --kc-policy cb --prefetch-threads 2 --gpus 0 --with-prior False"
in GTA-to-Cityscapes setting and find your code use  **validation set**(cityscapes) instead of training set(cityscapes) to produce pseudo-labels.
Does this accord with your experiment in eccv paper?

**Comment by Serge-weihao** [comment]
Could you explain why your code uses validation set of cityscapes for self-training and finally report the testing results on the same set? And did the ECCV paper follow the same process of this repo?

**Comment by yzou2** [comment]
Our ECCV paper follows the standard protocol the same as other works, using the labeled GTA5 and unlabeled Cityscapes **_train_** set to self-train the network, and test it on Cityscapes **_val_** set. We only use a subset. I think I made a mistake when uploaded the val list (The name is a little bit confusing). Sorry about that and thanks for pointing this out. And I uploaded the correct one.
